resources/memorytest/run.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+RESPONSE_FILE=response.txt
+
+#
+# The user must set these to valid values
+#
+LICENSE_FILE_PATH=
+ADMIN_PASSWORD=Password1
+if [ ! -f "$LICENSE_FILE_PATH" ]; then
+  echo 'Please supply the LICENSE_FILE_PATH parameter'
+  exit
+fi
+
+#
+# Build the valgrind image
+#
+echo 'Building the NGINX and valgrind Docker image ...'
+docker build --no-cache -t nginx_custom:v1 .
+if [ "$?" != '0' ]; then
+  echo "Problem encountered building the NGINX $DISTRO docker image"
+  exit
+fi
+
+#
+# Deploy the system
+#
+LICENSE_KEY=$(cat $LICENSE_FILE_PATH | jq -r .License)
+export LICENSE_KEY && export ADMIN_PASSWORD && docker compose up -d
+
+#
+# Wait for the Identity Server to come up
+#
+echo 'Waiting for the Curity Identity Server to start ...'
+c=0; while [[ $c -lt 25 && "$(curl -fs -w ''%{http_code}'' localhost:8443)" != "404" ]]; do ((c++)); echo -n "."; sleep 1; done
+
+#
+# Run curl tests to call the API via the reverse proxy
+#
+echo
+echo 'Running API tests ...'
+for TOKEN in 1 2 3 4 5
+do
+  #
+  # Act as a client to get a token
+  #
+  echo -n "."
+  HTTP_STATUS=$(curl -s -X POST http://localhost:8443/oauth/v2/oauth-token \
+    -H "Content-Type: application/x-www-form-urlencoded" \
+    -d "client_id=test-client" \
```

Comment:
should this client ID and secret be moved into variables that are set above, so it's easier to find and change?